x-pack/plugins/enterprise_search/public/applications/app_search/components/curations/curation/results/add_result_flyout.test.tsx

```diff
@@ -17,7 +17,7 @@ import { CurationResult, AddResultFlyout } from './';
 
 describe('AddResultFlyout', () => {
   const values = {
-    dataLoading: false,
+    searchDataLoading: false,
```

Comment:
I named this `searchDataLoading` rather than `dataLoading` so that it can exist alongside other logic files that may use `dataLoading` already.